components/odh-notebook-controller/controllers/notebook_controller_test.go

```diff
@@ -190,7 +190,7 @@ var _ = Describe("The Openshift Notebook controller", func() {
 		}
 
 		npProtocol := corev1.ProtocolTCP
-		testPodNamespace := Namespace
+		testPodNamespace := "redhat-ods-applications"
```

Comment:
This works now, thanks for looking at it.

I just wonder if this comment shouldn't be updated  as well

https://github.com/opendatahub-io/kubeflow/pull/90/files#diff-f43e081bcb07c8eb52ef6e4aeee0a508e129f48b0268f4b95e6c268ec9cdf648R223-R224

Reply:
Updated